Don't let lynx do domain name guessing. We want this configurable option
to be secure-by-default on Solaris.

This patch will not be submitted upstream.

--- lynx2-8-8/lynx.cfg.orig	2016-08-15 13:25:21.136798360 -0700
+++ lynx2-8-8/lynx.cfg	2016-08-15 13:26:53.913761192 -0700
@@ -678,6 +678,10 @@
 #
 #URL_DOMAIN_PREFIXES:www.
 #URL_DOMAIN_SUFFIXES:.com,.edu,.net,.org
+#
+# Provide a secure-by-default solution and don't do domain name guessing:
+URL_DOMAIN_PREFIXES:
+URL_DOMAIN_SUFFIXES:
 
 .h2 FORMS_OPTIONS
 # Toggle whether the Options Menu is key-based or form-based;
